tests/mocks/ERC1155_Receiver_mock.cairo

```diff
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: MIT
+
+%lang starknet
+
+from starkware.cairo.common.uint256 import Uint256
+from starkware.cairo.common.bool import TRUE, FALSE
+
+from openzeppelin.utils.constants import (
+    IERC1155_RECEIVER_ID, IACCOUNT_ID, ON_ERC1155_RECEIVED_SELECTOR, ON_ERC1155_BATCH_RECEIVED_SELECTOR 
+)
+
+@external
+func onERC1155Received(
+            operator : felt, from_ : felt, id : Uint256, amount : Uint256,
+            data_len : felt, data : felt*) -> (selector : felt):
+    if data_len == 0:
+        return (ON_ERC1155_RECEIVED_SELECTOR)
+    else:
+        return (FALSE)
+    end
+end
+
+@external
+func onERC1155BatchReceived(
+        operator : felt, from_ : felt, ids_len : felt, ids : Uint256*, 
+        amounts_len : felt, amounts : Uint256*, data_len : felt, data : felt*)
+        -> (selector : felt):
+    if data_len == 0:
+        return (ON_ERC1155_RECEIVED_SELECTOR)
```

Comment:
Hi, just saw that this return value should be `ON_ERC1155_BATCH_RECEIVED_SELECTOR` if I am not mistaken :)

Reply:
Correct, fixed :)